src/vtkIECTransformLogic.h

```diff
@@ -120,8 +121,8 @@ class VTK_IEC_TRANSFORM_LOGIC_EXPORT vtkIECTransformLogic : public vtkObject
   vtkTypeMacro(vtkIECTransformLogic, vtkObject);
   void PrintSelf(ostream& os, vtkIndent indent) override;
 
-  /// Update GantryToFixedReference transform based on gantry angle parameter
-  void UpdateGantryToFixedReferenceTransform(double gantryRotationAngleDeg);
+  /// Update GantryToFixedReference transform based on gantry rotation angle about the y-axis and gantry pitch angle about the x-axis parameter. The order of rotations is pitch rotation followed by gantry rotation
```

Comment:
in doxygen comment: specify that gantry pitch angle is not in IEC standard but rather a DICOM addition